### DISCUSSION TITLE
Is there an API to test whether the Server is currently functioning properly?

### DISCUSSION THREAD

**Discussion by kyousuke916** [pixiv/vroid-sdk-developers#87] [Q&A] [answered]
Hi, I'd like to ask if the VRoid SDK has a similar API that can check if the current Server is working properly, the idea is that if the VRoid Hub may be under maintenance and not functioning properly, it can be used to prompt players that they may not be able to get an account, character or character list, etc.

**Comment by nyamadan** [comment, marked as answer]
Hi.  there is currently no API that returns the server status.

If there are any incidents, we will announce them on [Twitter](https://twitter.com/vroid_pixiv_en).

**Reply by kyousuke916** [reply]
Got it, thanks.
